inf-ruby.el

```diff
@@ -505,10 +506,138 @@ Must not contain ruby meta characters.")
     (comint-send-string (inf-ruby-proc) (concat "\n" term "\n"))
     (when print (ruby-print-result))))
 
+(defface inf-ruby-result-overlay-face
+  '((((class color) (background light)) :background "black" :box (:line-width -1 :color "yellow")))
```

Comment:
I think we should just copy Cider's face definition here.

`grey90` is much lighter than `black` and should look better with light theme. `grey10` is probably better for dark as well.

Reply:
It this is fair, thank you for your suggestion @dgutov I've implemented that suggestion on this commit https://github.com/nonsequitur/inf-ruby/pull/153/commits/16c2acb4582961a3d45c8a208b0e09f4c54c2dcf.